backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java

```diff
@@ -72,6 +75,7 @@ private void buildUpgradesPayload(JsonObject aggregationPayload, JsonObject cont
     }
 
     private void buildServiceDisruptionPayload(JsonObject aggregationPayload, JsonObject context) {
+        String timestamp = aggregationPayload.getString(TIMESTAMP, LocalDateTime.now().toString());
```

Comment:
You may want to use this to avoid being dependent on the system default timezone:
```suggestion
        String timestamp = aggregationPayload.getString(TIMESTAMP, LocalDateTime.now(ZoneOffset.UTC).toString());
```